uswds/uswds
# USWDS - Nav: Update selector in escape unit test

## Summary

**Resolved nav unit test error.** Nav unit test for esacpe no longer causes an error. 

## Breaking change

This isn't a breaking change.

## Related issue

Closes #5081. 

## Preview link
Preview link: N/A

## Problem statement
Unit test was returning an error because it couldn't find a selector. 

## Solution
This updates the selected element when <kbd>Escape</kbd> it hit.

## Testing and review

1. Run `npm test:unit`
2. Go to `navigation toggle` test
3. Confirm there are no errors 


---
